mlflow/tensorflow/__init__.py

```diff
@@ -709,7 +723,7 @@ def _load_pyfunc(path):
         should_compile = save_format == "tf"
         K = importlib.import_module(keras_module.__name__ + ".backend")
         if K.backend() == "tensorflow":
-            K.set_learning_phase(0)
+            # K.set_learning_phase(0)
```

Comment:
why removing this line ? is the API changed ?

Reply:
This API doesn't exist anymore

Reply:
let's remove it.

Reply:
Yes, updated.